packages/components/src/styles/components/modal.scss

```diff
@@ -101,26 +58,19 @@
   width: min(800px, 95vw);
 }
 
-.hds-modal--color-neutral {
-  .hds-modal__header {
-    color: var(--token-color-foreground-strong);
-    background: var(--token-color-surface-faint);
-    border-bottom: 1px solid var(--token-color-border-primary);
-  }
-
-  .hds-modal__tagline {
-    color: var(--token-color-foreground-faint);
-  }
-}
+// Note ".hds-modal--color-neutral" uses default color and background-color styles from the dialog-primitive
 
 .hds-modal--color-warning {
   .hds-modal__header {
```

Comment:
If in the future we decide to yield also the `Description` in the Modal, this will happen:
<img width="651" alt="screenshot_3479" src="https://github.com/hashicorp/design-system/assets/686239/b6af64e7-827d-465c-9e08-a32ae4c94fad">

This tells me we should apply the background and border to the wrappers "containers", not the header/body/footer "content"

Reply:
Updated